marxan.cpp

```diff
@@ -2509,7 +2510,8 @@ void iterativeImprovement(int puno,int spno,vector<spustuff> &pu, vector<sconnec
         logBuffer << "iterativeImprovement after array init\n";
 
         // shuffle iimp array
-        random_shuffle(iimparray.begin(), iimparray.end());

```

Comment:
Interesting that this failed compilation. This was the function I used : http://www.cplusplus.com/reference/algorithm/random_shuffle/

Also I think there's already a seed value set in this file, but setting a new seed is fine too I think.

Reply:
https://en.cppreference.com/w/cpp/algorithm/random_shuffle
There it is listed as depreciated in C++14  and removed in in C++17. 
How to set new seed needs to be reviewed as future unit tests should produce  same results each time. 

template< class RandomIt >void random_shuffle( RandomIt first, RandomIt last ); | (1) | (deprecated in C++14)(removed in C++17)

  | (2) |  
template< class RandomIt, class RandomFunc >void random_shuffle( RandomIt first, RandomIt last, RandomFunc& r ); | (until C++11)
template< class RandomIt, class RandomFunc >void random_shuffle( RandomIt first, RandomIt last, RandomFunc&& r ); | (since C++11)(deprecated in C++14)(removed in C++17)
template< class RandomIt, class URBG >void shuffle( RandomIt first, RandomIt last, URBG&& g ); | (3) | (since C++11)
  |   |  